internal/pkg/cli/deploy/workload.go

```diff
@@ -380,7 +380,7 @@ func (d *workloadDeployer) uploadContainerImages(out *UploadArtifactsOutput) err
 		if err != nil {
 			return fmt.Errorf("generate docker build args: %w", err)
 		}
-		log.Infof(dockerengine.DockerBuildLabel(buildArgs.Platform, buildArgsList))
+		log.Infof(fmt.Sprintf("Building your container image: docker %s\n", strings.Join(buildArgsList, " ")))
```

Comment:
Is this because we want to show this log every single time when docker is building image?

Reply:
Looks like it removes a log dependency from dockerEngine 👍